src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -109,18 +109,29 @@ public IWatcher watch(String namespace, IOpenShiftWatchListener listener, String
 	}
 
 	@Override
+	public IWatcher watch(IOpenShiftWatchListener listener, String...kinds) {
+		return this.watch("", listener, kinds);
```

Comment:
@liggitt I have concerns about allowing users to watch the default namespace.  Doesnt this mean "watch for changes of the given type across all namespaces in the cluster"?  I recall this being 'expensive'